cmd/cli/upload.go

```diff
@@ -61,6 +61,7 @@ func UploadCmd(ctx context.Context, sigRef, payloadRef, imageRef string) error {
 		b64SigBytes, err = ioutil.ReadAll(os.Stdin)
 	} else if _, err = os.Stat(sigRef); os.IsNotExist(err) {
 		b64SigBytes = []byte(sigRef)
+		err = nil
```

Comment:
I think this is too tricky. I'd rather see this code cleaned up a bit to make it clearer what's going on.

Reply:
oof yea I apologize for this code.

Reply:
No worries!